assets/js/blocks/cart-checkout/checkout/style.scss

```diff
@@ -14,15 +14,15 @@
 
 .wc-block-checkout__actions {
 	display: flex;
-	flex-direction: column;
-	margin: 0 0 $gap-large*2;
-	padding-left: 9px;
+	justify-content: space-between;
+	align-items: center;
+	flex-direction: row-reverse;
```

Comment:
Why do we need to do this? Can't we change the order of the back to cart button and the checkout button in the markup and keep the default `flex-direction`?

Reply:
Yeah, absolutely. I went for a completely CSS solution but I can't see any reason to prefer that over changing the order of the buttons in the markup. Changed in 8a0d62af687124a0bba8d5b2eea14679d68c83a2.

Reply:
Nice, that's great. This was just a query about why you did it this way, in case I'd missed something that made this necessary. Thanks for the change :)